### DISCUSSION TITLE
Welcome to FlowPowerAppsMigrator Discussions!

### DISCUSSION THREAD

**Discussion by Zerg00s** [Zerg00s/FlowPowerAppsMigrator#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by keerthanar21** [comment]
Hello there,
I have followed the same as the instructions. But when I run the migrate-package.bat file it throws some kinda error.
Please help me to resolve this.
Many thanks in advance.

![image](https://user-images.githubusercontent.com/86395336/168432902-5a2c43cb-829f-4e93-828b-7b2bb426d6cd.png)

**Comment by Zerg00s** [comment]
Hi keerthanar21. 

It seems that `enum` feature that is used by the Flow&App migrator is a feature that requires PowerShell 5.0 later.

Would you be able to upgrade to PowerShell 5.0 or PowerShell 5.1?

https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/install/installing-windows-powershell?view=powershell-7.2

**Comment by fareedkhan27** [comment]
Hi Denis,

I was trying to test SharePoint list form PowerApps migration which have 2 lists from the same site. The below error is throwing, can you please help. Notes: I have downloaded latest version and Powershell too.

ForEach-Object : The property 'newId' cannot be found on this object. Verify that the property exists and can be set.
At C:\Users\khan\Downloads\FlowPowerAppsMigrator\CompleteResourceMapping.ps1:19 char:10
+ $lists | ForEach-Object {
+          ~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [ForEach-Object], RuntimeException
    + FullyQualifiedErrorId : PropertyAssignmentException,Microsoft.PowerShell.Commands.ForEachObjectCommand

**Comment by Zerg00s** [comment]
Hi @fareedkhan27,

Unfortunately, migration of forms within the same site is not supported yet. Since you are the 3rd person asking for this feature in the last 2 months, I might need to figure out how to get it working.

As a workaround, perhaps, you can migrate the form to a different site. I know, it's not ideal, but it's the only workaround currently available.

**Comment by cheney311** [comment]
I'm new to using FlowPowerAppsMigrator tool and was wondering if their are troubleshooting tips you'd recommend when errors are seen while running the tool. For instance a colleague of mine was running the tool to migrate a PowerApp from Src to Dst SharePoint site and received a Error The field was found invalid Apply-PnPProvisioningTemplate The field was found invalid: At C:\Users\userfirst.userlastname\Desktop\Transfer\MISC\Move-Lists.ps1:124 char:9, Apply-PnPProvisioningTemplate -Path Lists.xml. From what I see in the error it doesn't indicate what field/list etc that it is referring too. ANy troubleshooting advice would be appreciated. Thanks!

**Reply by Zerg00s** [reply]
Hi @cheney311,

I can try to help your colleagues. Then can: 
- search for a similar issue here: https://github.com/Zerg00s/FlowPowerAppsMigrator/issues?q=
- If the issue does not exist or is not helpful - create a new one. If possible, share the `"Lists.xml` file. I will try to apply it against my test/dev tenant.

**Comment by gillessylvestre** [comment]
What is the procedure for importing the PowarApps SharePoint form to the new site?

**Comment by Zerg00s** [comment]
hi @gillessylvestre ,
This is the step-by-step procedure: [Migrating Power Apps SharePoint List Form](https://github.com/Zerg00s/FlowPowerAppsMigrator#migrating-power-apps-sharepoint-list-form)

**Comment by gillessylvestre** [comment]
Good morning Denis. I hope this email finds you well. Thanks again for the great work.

Denis, I'm not a developer but I understand the concepts. I have a situation:

I have lists and field names containing special characters (/). Naturally, the script throws an error.

I know I have to escape the variables containing the names but I don't know which ones in which script.

Can you point me in the right direction, please?

Thank you.

Regards,

[cid:1253e341-a411-4fd2-b022-c689f777bdf2]
Gilles Sylvestre
1+ (819) 635-4958
